server/environment/environment.go

```diff
@@ -162,4 +162,6 @@ type Env interface {
 	SetSingleFlightDeduper(interfaces.SingleFlightDeduper)
 	GetPromQuerier() interfaces.PromQuerier
 	SetPromQuerier(interfaces.PromQuerier)
+	GetAuditLog() interfaces.AuditLog
+	SetAuditLog(log interfaces.AuditLog)
```

Comment:
could we call this GetAuditLogger/SetAuditLogger instead?

thinking logger is the object, log is the method we call on it